docs/getting-started/indy-walkthrough.md

```diff
@@ -35,9 +35,11 @@ If the concepts of cryptography and blockchain details feel mysterious, fear not
 
 Our goal is to introduce you to many of the concepts of Indy and give you some idea of what happens behind the scenes to make it all work.
 
-We're going to frame the exploration with a story. Alice, a graduate of the fictional Faber College, wants to apply for a job at the fictional company Acme Corp. As soon as she has the job, she wants to apply for a loan in Thrift Bank so she can buy a car. She would like to use her college transcript as proof of her education on the job application and once hired, Alice would like to use the fact of employment as evidence of her creditworthiness for the loan.
+**Please take note** that we are not going to cover peer-to-peer interaction as part of this tutorial. 
```

Comment:
See my previous comment. Saying we're not covering peer-to-peer makes no sense to me.